### PR TITLE
Single version for Fuel in an easy way

### PR DESCRIPTION
**Pull request by rizar**
Fixes the consequences of #254

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -18,9 +16,13 @@
 else:
     extra_compile_args = []
 
+exec_results = {}
+exec(open(path.join(path.dirname(__file__), 'fuel/version.py')), exec_results)
```
Comment (bartvm) [inline setup.py]: Nitpick, but I prefer using `with` instead of relying on the GC to close this file handle.

File: docs/conf.py [hunk 2]
```diff
@@ -92,9 +92,10 @@ def __getattr__(cls, name):
 # built documents.
 #
 # The short X.Y version.
-version = '0.0'
+import fuel
+version = fuel.__version__[:fuel.__version__.rindex('.')]
```
Comment (bartvm) [inline docs/conf.py]: Again, tiny nitpick, but if you don't want to assume that there are no more than 3 dots:
```
v[::-1].split('.', 1)[-1][::-1]
```
or
```
re.match(r'(.*?\..*?)\.', v).expand(r'\1')
```

File: setup.py (hunk 1 above)
Comment (rizar) [inline setup.py]: Makes sense, done.

File: docs/conf.py (hunk 2 above)
Comment (rizar) [inline docs/conf.py]: Check out yet another way to do it in the code.

Comment (bartvm) [inline docs/conf.py]: But 10 characters longer *and* O(n)! :open_mouth: 

Kidding aside, LGTM.

Comment (rizar) [inline docs/conf.py]: Sorry, I did try to find an exponential algorithm for that, but this is all I could do...

**Pull request merged**